package.json

```diff
@@ -63,38 +62,35 @@
 		"tmp": "0.2.1"
 	},
 	"devDependencies": {
-		"@types/jest": "29.5.0",
 		"@types/lodash": "4.14.191",
 		"@types/node": "20.8.8",
 		"@types/source-map-support": "0.5.6",
 		"@types/tmp": "0.2.3",
 		"@typescript-eslint/eslint-plugin": "5.56.0",
 		"@typescript-eslint/parser": "5.56.0",
+		"@vitest/coverage-istanbul": "0.34.2",
 		"eslint": "8.36.0",
 		"eslint-config-prettier": "8.8.0",
 		"eslint-config-strictest": "0.7.0",
 		"eslint-plugin-deprecation": "1.3.3",
 		"eslint-plugin-file-progress": "1.3.0",
 		"eslint-plugin-import": "2.27.5",
-		"eslint-plugin-jest": "27.2.1",
 		"eslint-plugin-node": "11.1.0",
 		"eslint-plugin-prettier": "4.2.1",
 		"eslint-plugin-promise": "6.1.1",
 		"eslint-plugin-unicorn": "46.0.0",
 		"genversion": "3.1.1",
-		"jest": "29.5.0",
-		"jest-environment-node": "29.5.0",
-		"jest-extended": "3.2.4",
-		"jest-mock-extended": "3.0.4",
 		"keep-a-changelog": "2.2.1",
 		"nodemon": "3.0.1",
 		"pm2": "5.2.2",
 		"prettier": "2.8.6",
 		"prettier-plugin-prisma": "4.10.0",
 		"prisma": "4.3.1",
 		"semver": "7.5.4",
-		"ts-jest": "29.0.5",
 		"ts-node": "10.9.1",
-		"typescript": "4.9.5"
+		"typescript": "4.9.5",
+		"undici": "5.23.0",
```

Comment:
`undici` is redundant here; now that we're on Node 20, `fetch` is a built-in global.

Reply:
Whoops, I removed that on the last PR but apparently it didn't merge over right 🤦‍♂️